metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -135,12 +135,8 @@ export type InstrumentationConfigMap = {
 export function getNodeAutoInstrumentations(
   inputConfigs: InstrumentationConfigMap = {}
 ): Instrumentation[] {
-  for (const name of Object.keys(inputConfigs)) {
-    if (!Object.prototype.hasOwnProperty.call(InstrumentationMap, name)) {
-      diag.error(`Provided instrumentation name "${name}" not found`);
-      continue;
-    }
-  }
+  checkManuallyProvidedInstrumentationNames(Object.keys(inputConfigs));
+  const instrumentationsFromEnv = getAllowedInstrumentationsFromEnv();
```

Comment:
```suggestion
  const enabledInstrumentationsFromEnv = getEnabledInstrumentationsFromEnv();
```

Reply:
renamed with [this commit](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1953/commits/5f7a4cf6698b210bbc59e65fcda6e4c05e73b804), that is squashed into [the feature commit](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1953/commits/c7393d9ec7dddbdc7031d37a420d69fd30e11614) after rebasing on top of latest main